### ISSUE TITLE
I can't unsign

### ISSUE THREAD

**Issue by Security-Xenc** [Paradoxis/Flask-Unsign#2]
I try secret_key into all.txt , but use flask-unsign is failed, parm --secret use 5201314 is wrong,tips int?

**Comment by Paradoxis** [comment]
Have you followed the troubleshooting guide in tehe readme?

**Comment by Paradoxis** [comment]
I just added an issue template for you, could you please fill in the following? It would help debugging the issue, thanks!

## Stack Trace
If you got a stack trace, please paste it here:

```
...
```

## To Reproduce
Please add the following files / info:

- The session cookie you wish to crack (or one which can reproduce the issue)
- The password file you're using (unless it's the built-in wordlist)
- The password you know the session is signed with

## System information
Please add the output of the following commands:

### Flask-Unsign Version
```
$  pip3 freeze | grep -i flask-unsign
...
```

### Flask-Unsign Type
```
$ file $(which flask-unsign)
...
```

### Python Version
```
$ python3 --version
...
```

### System Version
```
$ uname -a
...

$ cat /etc/issue
...
```

## Screenshots
A picture tells a thousand words, feel free to add one here if you have it

## Other stuff
Got any other info I should know about? Have a monkeypatch or proposed fix?  <br>
Feel like ranting about something? Feel free to do it here. Compliments are also welcome ✨

**Comment by Security-Xenc** [comment]
[!] Aborted. ？ 
What is the meaning of this

**Comment by Paradoxis** [comment]
It’s printed whenever you raise a KeyboardInterrupt (a CTRL+C), but looking at the error you previously mentioned it might be possible you have to either wrap the secret with additional quotes or use the --no-literal-eval argument as the string is interpreted as an int (this is because some binary strings have to fit in a ascii wordlist as well without corrupting the file when a \n character appears)

Could you still fill in the template for me so I have more context?

**Comment by Paradoxis** [comment]
Reproduced the bug for you, the full stack trace would have been nice to include. Looking for a fix now

```
(venv) paradoxis-base:flask-unsign Paradoxis$ flask-unsign --sign --secret '126848' --cookie "{'hello': 'world'}"
Traceback (most recent call last):
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/venv/bin/flask-unsign", line 11, in <module>
    load_entry_point('flask-unsign', 'console_scripts', 'flask-unsign')()
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/flask_unsign/helpers.py", line 78, in wrapper
    return func() or 0
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/flask_unsign/__main__.py", line 182, in main
    legacy=args.legacy
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/flask_unsign/session.py", line 43, in sign
    return get_serializer(secret, legacy, salt).dumps(value)
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/venv/lib/python3.6/site-packages/itsdangerous/serializer.py", line 167, in dumps
    rv = self.make_signer(salt).sign(payload)
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/venv/lib/python3.6/site-packages/itsdangerous/timed.py", line 42, in sign
    return value + sep + self.get_signature(value)
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/venv/lib/python3.6/site-packages/itsdangerous/signer.py", line 143, in get_signature
    key = self.derive_key()
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/venv/lib/python3.6/site-packages/itsdangerous/signer.py", line 132, in derive_key
    mac = hmac.new(self.secret_key, digestmod=self.digest_method)
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/venv/lib/python3.6/hmac.py", line 144, in new
    return HMAC(key, msg, digestmod)
  File "/Users/Paradoxis/Documents/Projects/flask-unsign/venv/lib/python3.6/hmac.py", line 42, in __init__
    raise TypeError("key: expected bytes or bytearray, but got %r" % type(key).__name__)
TypeError: key: expected bytes or bytearray, but got 'int'
```

**Comment by Paradoxis** [comment]
As it turns out you had to use the `--no-literal-eval` argument, I'll make a patch which makes this a little more user friendly.

**Comment by Paradoxis** [comment]
Fixed in version 0.0.2
